File: src/main/java/com/oshi/libgenericsettings/data/ExpandableTitleSubtitleSimpleItemsData.java
package com.oshi.libgenericsettings.data;

import android.support.annotation.ColorRes;

import java.util.List;


public class ExpandableTitleSubtitleSimpleItemsData extends ExpandableTitleSimpleItemsData {

    protected String subtitle;
    protected @ColorRes int subtitleColor;

    public ExpandableTitleSubtitleSimpleItemsData(String title, String subtitle, List<String> items) {
        super(title, items);
        this.subtitle = subtitle;
    }

    public int getSubtitleColor() {
        return subtitleColor;
    }

    public void setSubtitleColor(int subtitleColor) {
        this.subtitleColor = subtitleColor;
    }

    public String getSubtitle() {
        return subtitle;
    }

    public void setSubtitle(String subtitle) {
        this.subtitle = subtitle;
    }

    @Override
    public int getViewType() {
        return VIEW_TYPE_SETTINGS_EXPANDABLE_TITLE_SUBTITLE_SIMPLE_ITEMS;
    }
}

File: src/main/java/com/oshi/libgenericsettings/data/ExpandableTitleSubtitleSimpleItemsData.kt
package com.oshi.libgenericsettings.data

import android.support.annotation.ColorRes

/**
 * Expandable Title & Subtitle with Simple (String) sub items. Being use in layout/view_type_expandable_title_subtitle_simple_items.xml
 */
class ExpandableTitleSubtitleSimpleItemsData(title : String, var subtitle: String, items : List<String>) : ExpandableTitleSimpleItemsData(title, items) {

    @ColorRes var subtitleColor : Int = 0


    override fun getViewType(): Int {
        return VIEW_TYPE_SETTINGS_EXPANDABLE_TITLE_SUBTITLE_SIMPLE_ITEMS
    }
}